ouroboros-network type checking error on Windows

```
src\Ouroboros\Network\Diffusion.hs:220:10: error:
    * Couldn't match type `SockAddr' with `[Char]'
      Expected type: Versions
                       NodeToClientVersion
                       DictVersion
                       (ConnectionId LocalAddress
                        -> OuroborosApplication 'ResponderApp ByteString IO Void ())
        Actual type: Versions
                       NodeToClientVersion
                       DictVersion
                       (ConnectionId SockAddr
                        -> OuroborosApplication 'ResponderApp ByteString IO Void ())
    * In the fifth argument of `NodeToClient.withServer', namely
        `(daLocalResponderApplication applications)'
      In the expression:
        NodeToClient.withServer
          sn
          (NetworkServerTracers
             dtMuxLocalTracer dtHandshakeLocalTracer dtLocalErrorPolicyTracer)
          networkLocalState
          (Snocket.localAddressFromPath daLocalAddress)
          (daLocalResponderApplication applications)
          localErrorPolicy
      In an equation for `runLocalServer':
          runLocalServer sn networkLocalState
            = NodeToClient.withServer
                sn
                (NetworkServerTracers
                   dtMuxLocalTracer dtHandshakeLocalTracer dtLocalErrorPolicyTracer)
                networkLocalState
                (Snocket.localAddressFromPath daLocalAddress)
                (daLocalResponderApplication applications)
                localErrorPolicy
    |
220 |         (daLocalResponderApplication applications)
    |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabal.exe: Failed to build ouroboros-network-0.1.0.0 (which is required by
```

--- a/ouroboros-network/src/Ouroboros/Network/Diffusion.hs
+++ b/ouroboros-network/src/Ouroboros/Network/Diffusion.hs
@@ -110,7 +110,7 @@ data DiffusionApplications = DiffusionApplications {
     , daLocalResponderApplication :: Versions
                                        NodeToClientVersion
                                        DictVersion
-                                       (ConnectionId SockAddr ->
+                                       (ConnectionId LocalAddress ->
                                           OuroborosApplication
                                             ResponderApp
                                             ByteString IO Void ())

